operator/controllers/csiscaleoperator_controller.go

```diff
@@ -468,7 +470,16 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, err
 	}
 
-	csiNodeSyncer := clustersyncer.GetCSIDaemonsetSyncer(r.Client, r.Scheme, instance, daemonSetRestartedKey, daemonSetRestartedValue, CGPrefix)
+	cm, err := r.getConfigMap(instance, config.CSIEnvVarConfigMap)
+	if err != nil {
+		logger.Error(err, "Failed to fetch configMap details from the cluster.", "configMap", config.CSIEnvVarConfigMap)
+		return ctrl.Result{}, err
+	}
+
+	cmData := map[string]string{}
+	cmData = parseConfigMap(cm, config.CSINode.String())
```

Comment:
only a parameter `cm` should be sufficient, right? Is there any specific reason to pass 2nd param here?